src/tests/ftest/util/nvme_utils.py

```diff
@@ -92,6 +95,16 @@ def setUp(self):
         self.engines = self.server_managers[0].manager.job.yaml.engine_params
         self.dmg_command = self.get_dmg_command()
 
+    def create_container(self):
+        """Create the container """
+        self.nvme_local_cont = self.get_container(self.pool, create=False)
+	
+        # update container oclass
+	if self.ior_local_cmd.dfs_oclass:
```

Comment:
(pylint-syntax-error) inconsistent use of tabs and spaces in indentation (<unknown>, line 103)